src/board.js

```diff
@@ -0,0 +1,73 @@
+import { useState } from 'react'
+import { Square } from './square.js'
+
+const board = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
```

Comment:
MAYBE move this into a constants file for other future variables

Reply:
Just a file with a bunch of `export const board = ...`?

Reply:
Yea you can make it fancier later but having one place for constants is nice so you dont have to go back through later or forget about stuff